### ISSUE TITLE
Error: i18n support is not compatible with next export.

### ISSUE THREAD

**Issue by Chhunneng** [vinissimus/next-translate#883]
I use next-translate in my project, when deployment I need to run `next build && next export -o dist` but I got error related to `i18n`.
```command
Error: i18n support is not compatible with next export. See here for more info on deploying: https://nextjs.org/docs/deployment
    at /home/ubuntu/Desktop/orkun-admin-site-2/node_modules/next/dist/export/index.js:149:19
    at async Span.traceAsyncFn (/home/ubuntu/Desktop/orkun-admin-site-2/node_modules/next/dist/trace/trace.js:79:20)
error Command failed with exit code 1.
```

**Comment by aralroca** [comment]
Yes, next-translate uses the i18n routing implementation of Next.js, which is implemented in Next.js core and has this limitation:

https://nextjs.org/docs/advanced-features/i18n-routing#how-does-this-work-with-static-generation

From here we can't stand it. I close the issue and you should report the issue in the Next.js repository so they can implement it if they see that many people need it.

Next-translate is more focused on translation, but i18n routing is part of Next.js core.

**Comment by Benji377** [comment]
I just got this issue while trying to build a Tauri application with Next.js as frontend. I noticed that there is already an open discussion about this topic in the [Next.js discussions](https://github.com/vercel/next.js/discussions/18957). I just wanted to know if I could use the same fixes suggested there with this package too?
Adrian wrote a Blog post about [how to fix this issue](https://dev.to/adrai/static-html-export-with-i18n-compatibility-in-nextjs-8cd), do the changes proposed here also work with the next-translate plugin? @aralroca
